File: beginners/easy/002.positive-negative/206.saadman-galib.positive-negative.cpp
#include<iostream>
using namespace std;

int main(){
    double num;
    cout << "Enter your input: ";
    cin >> num;

    if(num > 0){
        cout << "This is a positive number" << endl;
    }
    else if (num < 0){
        cout << "This is a negative number" << endl;

    }
    else{
        cout << "This is zero" << endl;
    }

    return 0;
}

File: beginners/easy/002.positive-negative/206.saadman-galib.positive-negative.go
package main
import "fmt"

func main(){
	var num float64
	fmt.Scanln(&num)

	if num > 0 {
		fmt.Println("This is a positive number")
	} else if num < 0 {
		fmt.Println("This is a negative number")
	} else {
		fmt.Println("This is zero")
	}
}